### PR TITLE
Ajout possibilité de sortir par le trou

### PR DESCRIPTION
**Pull request by mdan312**
Ajout possibilité de sortir par le trou
Ça fait du bien de pouvoir respirer le grand air!!!

### CODE REVIEW

**Comment by mdan312** [comment]
@casey-collab

File: french/manger/faim.md [hunk 1]
```diff
@@ -4,3 +4,5 @@ commencer la journée ! ;)
 A force de manger vous faites un trou dans le mur
 
 [Je sors](../sortie/chercher-la-sortie.md)
+[Je me faufille par le trou](trou/trou.md)
```
Comment (casey-collab) [inline french/manger/faim.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by mdan312** [comment]
t'es pénible @casey-collab !

**Comment by casey-collab** [comment]
Looks great! :octocat:

**Pull request merged**